ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/host_vars/fabric_numbering_id.yml

```diff
@@ -0,0 +1,11 @@
+type: spine
+
+spine:
+  defaults:
+    bgp_as: 65001-65100
+    loopback_ipv4_pool: 192.168.0.0/24
+    # id: 1 will be set by pool_manager
+
+fabric_numbering:
+  id:
+    algorithm: pool_manager
```

Comment:
If ppl want to split and use fabric_numbering for a subset of devices, I think we can accept that. Ex. only use it for your l2leafs or something. We catch the issue of manual id being set together with pool-manager on one device.
1. It is very hard to catch this.
2. I am not sure we really want to prevent this. Being open for a step-wise introduction of pool-manager is good IMO.
3. We also accept overlapping IDs today, since ppl might have custom algorithms.

Reply:
I agree with Claus' comments.